amaechi-chuks/Politico
# #163682325 Update user profile page

**What does this Pr do?**
* Implement image uploads
* differentiate user and admin on login session

#### Description of tasks to be completed

    * add admin login sesssion

    * add user login session

    * protect secret key by pipping into .env file

    * refactor code base


#### How should this be manually tested

    * Clone the repo: Run git clone https://github.com/yomigeek/mtracker-react.git in the folder where you want the repo to be saved.

    * Navigate into the newly created folder and install the dependencies: cd folder_name && npm install

    * Start the server using npm run start:dev

    * Make a Post request to http://localhost:6000/auth/signup
  
    * You get a signup page for the application

    * Provide your details and click submit, You will be redirected to user profile page

   * Log out by clicking the log out button

    * Make a POST request to http://localhost:6000/auth/login

    * You get a login page for the application

    * Login wit your email and password and see how magically the sysytem will redirect you to your user profile details 

    * Login with an existing admin account (username:[amaechichuks2000@gmail.com, password:fabulous26), the dashboard page displays

    * Log out and login and login again and you will be redirected to admin profile page


#### What are the necessary pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163682325
[Delivers|#163682325]
